docs/theming.mdx

```diff
@@ -38,21 +53,71 @@ render(
 ### styled
 
 ```jsx
-// @live
 /** @jsx jsx */
 import { jsx } from '@emotion/core'
+import { useTheme } from './theme'
+
+let Button = props => {
+  let theme = useTheme()
+  return (
+    <button
+      css={{
+        color: theme.colors.primary
+      }}
+      {...props}
+    />
+  )
+}
+
+let SecondaryButton = props => {
+  let theme = useTheme()
+  return (
+    <button
+      css={{
+        color: theme.colors.secondary
+      }}
+      {...props}
+    />
+  )
+}
+```
+
+Or with the styled api like this
+
+```jsx
 import styled from '@emotion/styled'
-import { ThemeProvider } from 'emotion-theming'
+import { useTheme } from './theme'
 
-const theme = {
-  colors: {
-    primary: 'hotpink'
+let Button = styled.button(() => {
+  let theme = useTheme()
```

Comment:
This usage and the below usage trigger errors in the latest React. 

```console
Uncaught Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.
```

Reply:
@stramel have you tried to use this with emotion 10 or 11? It can only work with the latter.

Reply:
Oh let me try! Thank you for the quick reply!

Reply:
@Andarist I'm running it inside Docz v2 using `@emotion/styled` and `@emotion/core` on version `11.0.0-next.9`. 

When starting the project it complains because it can't find `@emotion/styled-base`. 

```console
ERROR #98123  WEBPACK

Generating SSR bundle failed

Can't resolve '@emotion/styled-base' in 'Projects/libs/atoms/src/lib/Button'
```

so I installed `@emotion/styled-base` version `v11.0.0-next.3` and it is telling me i need to use `@emotion/styled`? 

When I tried the `v10` version of `@emotion/styled-base` it obviously failed... 😞

Reply:
Seems like maybe you have 2 versioms of emotion in your node modules for some reason or that you are using v10 of our babel plugin. `@emotion/styled-base` has been removed in v11 in favor of `@emotion/styled/base`.

If you prepare a runnable repro case then I could take a look at it.

Reply:
Alright, let me do some debugging on what you mentioned might be the problem before I make to you chase a ghost.

Reply:
@Andarist Looks like you were correct. Just tried it in my Next.js app with v11 and no errors. Doing some debugging...

`yarn why @emotion/core` and `yarn why @emotion/styled` both result in v10 and v11 because Docz and the gatsby docz theme both use v10

Reply:
Any suggestions how to fix it?

Reply:
I filed an issue on docz asking for a bumping to v11 but I'm not sure what the workaround would be. https://github.com/doczjs/docz/issues/1335

Reply:
Thank you @stramel! Anyone know the expected release date of v11?

Reply:
I can't promise anything, but hopefully within a month. We have most of the things done, some final touches left, but this is a work we are doing in our free time so there is also a possibility that this will take some more time.